Program.fs

```diff
@@ -37,6 +36,11 @@ type ResolveAction =
     | UninstallDev of library:string * package: string * version:string
     | UnableToResolve of library:string * package:string * range:string * error: string
 
+type RawPackage = {
```

Comment:
Suggestion: rename `RawPackage` to `PackageJson` because these fields are read from a package.json file